README.md

```diff
@@ -210,23 +204,18 @@ Be sure to escape closing `</script>` tags within your source as `<&lt;script>`.
 </playground-project>
 ```
 
-### Option 2: JSON manifest
-
-Serve a JSON file containing a `files` object. Keys are filenames relative to
-the manifest URL. Values are objects with any of the following optional
-properties:
-
-- `content`: Optional text content of the file. If omitted, a `fetch` is made to
-  retrieve the file by filename, relative to the manifest URL.
-
-- `contentType`: Optional MIME type of the file. If omitted, type is taken from
-  either the `fetch` response `Content-Type` header, or inferred from the
-  filename extension when `content` is set.
+### Option 2: JSON configuration
 
-- `label`: Optional label for display in `playground-tab-bar`. If omitted, the
-  filename is displayed.
+Set the `project-src` attribute or `projectSrc` property to a JSON file with format:
 
-- `hidden`: If `true`, the file won't be visible in `playground-tab-bar`.
+| Property            |  Description                                                                                                                                                                     |
+| ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `files`             | An object mapping filenames to file data.                                                                                                                                        |
+| `files.content`     | Optional text content of the file. If omitted, a `fetch` is made to retrieve the file by filename, relative to the manifest URL.                                                 |
+| `files.contentType` | Optional MIME type of the file. If omitted, type is taken from either the `fetch` response `Content-Type` header, or inferred from the filename extension when `content` is set. |
+| `files.label`       | Optional label for display in `playground-tab-bar`. If omitted, the filename is displayed.                                                                                       |
+| `files.hidden`      | If `true`, the file won't be visible in `playground-tab-bar`.                                                                                                                    |
+| `extends`           | Optional URL to another JSON config file to extend from. Configurations are deeply merged.                                                                                       |
```

Comment:
Is this relative or absolute and relative to what? (is this even an applicable question I'm asking?)

Reply:
Updated the comment, is it clear? Was surprisingly hard to write a sentence that specifies it. I think it's the intuitive behavior (the URL is relative to the place where you wrote the URL).